src/main/java/com/exasol/dbbuilder/dialects/exasol/ExasolImmediateDatabaseObjectWriter.java

```diff
@@ -31,18 +35,13 @@ public ExasolImmediateDatabaseObjectWriter(final Connection connection) {
      * @param adapterScript the adapter script to be created
      */
     // [impl->dsn~creating-adapter-scripts~1]
-    // [impl->dsn~creating-adapter-scripts-with-debugger~1]
+    // [impl->dsn~creating-java-scripts-with-jvm-options~1]
     public void write(final AdapterScript adapterScript) {
         final StringBuilder sqlBuilder = new StringBuilder("CREATE " + adapterScript.getLanguage() + " ADAPTER SCRIPT "
                 + adapterScript.getFullyQualifiedName() + " AS\n" + adapterScript.getContent() + "\n");
-
-        if (ExasolConfiguration.getInstance().isAdapterScriptDebuggingEnabled()) {
-            if (!adapterScript.hasDebuggerConnection()) {
-                throw new IllegalStateException("Debugging is enabled but debug connection is missing. "
-                        + "You can either disable debugging or provide a debug connection with the AdatperScript creation.");
-            }
-            final String debuggerConnection = adapterScript.getDebuggerConnection();
-            sqlBuilder.append("%jvmoption -agentlib:jdwp=transport=dt_socket,server=n,address=" + debuggerConnection+";\n");
+        if (adapterScript.getLanguage().equals(AdapterScript.Language.JAVA)
+                && !this.configuration.getJvmOptions().isEmpty()) {
+            sqlBuilder.append("%jvmoption " + String.join(" ", this.configuration.getJvmOptions()) + ";\n");
```

Comment:
I usually put these %jar, %jvmoption as the first lines in the udf creation, but I think it does not matter after the content, right?

Reply:
Ich changed it.